test/test_helper.rb

```diff
@@ -30,6 +30,38 @@
   ActiveRecord::Migration.execute "SET CONSTRAINTS ALL DEFERRED;"
 end
 
+module TestCaseRailsLoggerExtensions
+  def before_setup
+    super
+    @current_buffer = StringIO.new
```

Comment:
`@current_buffer` sounds too generic and can lead to conflicts with tests (if they used current_buffer for something. Could you thing of more specific name?